tests/components/tedee/test_config_flow.py

```diff
@@ -40,6 +40,7 @@ async def test_flow(hass: HomeAssistant, mock_tedee: MagicMock) -> None:
     assert result2["data"] == {
         CONF_HOST: "192.168.1.62",
         CONF_LOCAL_ACCESS_TOKEN: "token",
+        CONF_WEBHOOK_ID: result2["data"][CONF_WEBHOOK_ID],
```

Comment:
I'd rather see the webhook generate id patched and then check if that is correctly saved